contracts/common/test_helpers/Assertions.sol

```diff
@@ -0,0 +1,214 @@
+// SPDX-FileCopyrightText: 2023 Lido <info@lido.fi>
+// SPDX-License-Identifier: GPL-3.0
+
+/* See contracts/COMPILERS.md */
+pragma solidity 0.8.9;
```

Comment:
not sure it should be placed in the `common` though

Reply:
yeah the tests are using 0.8.9 and that's the problem. however, the `MemUtils` itself supports 0.4.24

Reply:
not sure what to do here. should we move the tests to `0.8.9` given that the tested code is in `common`?

Reply:
writing assertions supporting 0.4.24 would be a pain

Reply:
I guess we can proceed as is, too minorish for me